Cubical/Structures/Maybe.agda

```diff
@@ -80,6 +83,11 @@ module MaybePathP where
     isom .Iso.rightInv = decodeEncode ox oy
     isom .Iso.leftInv = encodeDecode A ox oy
 
+-- Structured isomorphisms
+
+maybe-structure : (S : Type ℓ → Type ℓ₁) → Type ℓ → Type ℓ₁
+maybe-structure S X = Maybe (S X)
```

Comment:
This is the kind of thing I was thinking would be a composition of structures. Maybe is functorial structure and S is just a structure... Could this make sense?

Reply:
My intuition is that it won't make sense, but I'll think about it.